HTML/exercise-05/script.js

```diff
@@ -0,0 +1,220 @@
+  )
+    .then((res) => {
+      res.json().then((r) => {
+        if (r.Response === "True") {
+          // console.log(r,);
+          movies = r.Search;
+          console.log(r);
+          found = true;
+          if (parseInt(r.totalResults) > 10) {
+            paginationDiv.style.display = "flex";
+
+            sortingDivCont.style.display = "flex";
+            currettext = textinput.value;
+            paginationP.innerText =
+              page + " / " + Math.round(parseInt(r.totalResults) / 10);
+          }
+          while (cardCont.firstChild) {
+            cardCont.removeChild(cardCont.firstChild);
+          }
+          Sort();
+        } else {
+          alert(r.Error);
+        }
+      });
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+}
+
+function createCard(data, index) {
+  var ele = document.createElement("div");
+  ele.setAttribute("class", "card");
+  // creating img div
+  var imgcont = document.createElement("div");
+  imgcont.setAttribute("class", "imgCont");
+  var imgnode = document.createElement("img");
+  imgnode.setAttribute("src", data.Poster);
+  imgnode.setAttribute("alt", data.Title);
+  imgcont.appendChild(imgnode);
+
+  //creating title div
+  var titlediv = document.createElement("div");
+  titlediv.setAttribute("class", "desc");
+  var titlep = document.createElement("p");
+  titlep.innerText =
+    "Title : " +
+    (data.Title.length > 20 ? data.Title.slice(0, 20) + "..." : data.Title);
+  titlediv.appendChild(titlep);
+
+  //creating year div
+  // var yeardiv = document.createElement("div")
+  // yeardiv.setAttribute("class", "desc")
+  var yearp = document.createElement("p");
+  yearp.innerText =  data.Year;
+  yearp.style.textAlign = "end"
+  titlediv.appendChild(yearp);
+
+  ele.appendChild(imgcont);
+  ele.appendChild(titlediv);
+
+  ele.setAttribute("key", data.imdbID);
+
+  ele.onclick = () => {
+    posterClick(data.imdbID);
+  };
+  cardCont.appendChild(ele);
+}
+
+function getPage(pageNo) {
+  // paginationDiv.style.display = "none";
+  fetch(
+    "http://www.omdbapi.com/?apikey=fd717b92&type=movie&s=" +
+      currettext +
+      "&page=" +
+      pageNo
+  )
+    .then((res) => {
+      res.json().then((r) => {
+        if (r.Response === "True") {
+          // console.log(r,);
+          movies = r.Search;
+          console.log(r);
+          found = true;
+          if (parseInt(r.totalResults) > 10) {
+            //   paginationDiv.style.display = "flex";
+            currettext = textinput.value;
+            paginationP.innerText =
+              page + " / " + Math.round(parseInt(r.totalResults) / 10);
+          }
+          while (cardCont.firstChild) {
+            cardCont.removeChild(cardCont.firstChild);
+          }
+          Sort();
+        } else {
+          alert(r.Error);
+        }
+      });
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+}
+
+sortbyId.onchange = Sort;
+sortOrderId.onchange = Sort;
+function Sort() {
```

Comment:
use the camelCase naming convention here and also name the your functions and variables to clearly indicate their purpose